src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -308,6 +308,18 @@ begin
   simp [*],
 end
 
+@[simp]
+lemma walk.subset_support_append_left {V : Type u} {G : simple_graph V} {u v w : V}
+  (p : G.walk u v) (q : G.walk v w) :
+  p.support ⊆ (p.append q).support :=
+by simp only [walk.support_append, list.subset_append_left]
+
+@[simp]
+lemma walk.subset_support_append_right {V : Type u} {G : simple_graph V} {u v w : V}
+  (p : G.walk u v) (q : G.walk v w) :
+  q.support ⊆ (p.append q).support :=
+by { intro h, simp only [mem_support_append_iff, or_true, implies_true_iff] {contextual := tt}}
```

Comment:
I think the usual style is to include these spaces (although squeeze_simp doesn't produces them).
```suggestion
by { intro h, simp only [mem_support_append_iff, or_true, implies_true_iff] { contextual := tt }}
```
By the way, you don't need to squeeze "terminal simps" (simps that are the last tactic). There's no harm in doing it though. I sometimes do it when the simp is slow, or if there's only a handful of lemmas involved (like here).